clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -210,11 +234,18 @@ export class CopybookDownloadService implements vscode.Disposable {
         while (this.queue.length > 0) {
             toDownload.push(await this.queue.pop());
         }
-        toDownload.filter(cp => !cp.quiet).map(cp => cp.copybook).forEach(cb => errors.add(cb));
+        toDownload.map(cp => cp.copybook).forEach(cb => errors.add(cb));
```

Comment:
working on it

Reply:
So as discussed with @zimlu02 ,  and the initial idea was - 
`(either intently or because he didn’t know the right one) didn’t provide right DSN path or profile he got spammed with error messages every-time he opened any cobol file - it was extremely annoying`

So, we will ignore all messages download_err_msg, which we generate, but will keep showing the response from zone API errors (even in quiet mode). So suppose, there is a cred err, we will display it in quiet mode as well. 
But if we can't find a copybook due to the wrong path setting. we won't show them in quiet mode.
@grianbrcom What do you think?

Reply:
Looks like we need to keep the current way. Don't show anything in a quiet mode, even auth errors. But in case of auth error we "lock" the profile and don't try again. And only when the user manually starts downloading process, we show the message that the profile was locked (but didn't try to connect again).

Reply:
this error set is updated later to include only the non-quite copybook req. I used this errors set to get info, on what copybooks need to try for USS download.